x-pack/plugins/observability_solution/synthetics/public/apps/synthetics/components/monitor_add_edit/monitor_add_page.tsx

```diff
@@ -33,6 +34,8 @@ export const MonitorAddPage = () => {
 
   const canUsePublicLocations = useCanUsePublicLocations();
 
+  const { data: cloneMonitor, loading } = useCloneMonitor();
```

Comment:
```suggestion
  const { data: cloneMonitor, loading: cloneMonitorLoading } = useCloneMonitor();
```

We've got multiple loading states tracked in this component, so please label this one more specifically.